docs/jobs.md

```diff
@@ -96,6 +96,41 @@ primaryJob: example
 
 This is done by simply using the name of one of the jobs in the `primaryJob` attribute of the config file. This will allow Skelebot to set this job as the default command for the docker image that is built, thereby making a more easily distributable Docker Image for the sake of deployment.
 
+#### Primary Exe
+The `primaryExe` field in the skelebot.yaml config allows for the specification of an execution command to use in the Dockerfile. This field accepts two different values: "ENTRYPOINT" and "CMD". If the field is not specified in the config, the default "CMD" value is used.
+
+```
+primaryExe: (ENTRYPOINT, CMD)
+```
+
+Using "CMD" as the primary execution method requires that the primary job is configured to use only parameters, not arguments, and that each parameter has a default value. This allows the command string can be constructed in full so that it can be run without any extra parameters in the Docker Run command.
```

Comment:
```suggestion
Using "CMD" as the primary execution method requires that the primary job is configured to use only parameters, not arguments, and that each parameter has a default value. This allows the command string to be constructed in full so that it can be run without any extra parameters in the Docker Run command.
```